src/components/Doc/ComponentDoc.js

```diff
@@ -79,6 +87,41 @@ export const ComponentDoc = ({
         )}
       </Box>
 
+      {designSystemLink && (
+        <ThemeContext.Extend
+          value={{
+            notification: {
+              container: {
+                background: {
+                  color: 'background-contrast',
+                },
+                round: 'small',
+                pad: 'small',
+              },
+              direction: 'row',
+              message: {
+                color: 'text-strong',
+              },
+            },
+          }}
+        >
+          <Notification
+            message="Using Grommet for HPE?"
+            status="info"
+            actions={[
+              {
+                label: `See ${name} for examples and guidance.`,
+                href: designSystemLink,
+                color: 'text-strong',
+                style: {
```

Comment:
@ericsoderberghp I know you aim to avoid inline styling as much as possible, but this feels like the cleanest to getting an underline under the text. Are you okay with this?

Reply:
Something just feels existentially wrong to me about having inline styles in the Design System site. Can we just start with the default and see how it looks?

Reply:
Yes, we can do that. @JCoder121 -- let's remove the inline style of underline and custom text color and just leave as the default grommet anchor treatment.